server/models/User.py

```diff
@@ -88,18 +99,21 @@ class UserSchema(Schema):
     password = fields.String(
         required=True, validate=validate.Length(min=6), load_only=True)
     confirmPassword = fields.String()
+
+    address = fields.String()
+    generalLocation = fields.String()
```

Comment:
Did you and @mssiegel agree on the database fields for this? Are we removing lat and long from the database? I believe we will need that to actually do the searching

Reply:
yeah, gabriel and i spoke and agreed to these fields. the latitude and longitude are still in the user model

Reply:
latitude and longitude should stay in the user schema because they're necessary for the google map on user profile page